web/src/search_suggestion.ts

```diff
@@ -752,42 +754,74 @@ function get_operator_suggestions(last: NarrowTerm): Suggestion[] {
     });
 }
 
+// One search suggestion can include multiple search terms, based on what's
+// in the search bar.
+type SuggestionLine = Suggestion[];
```

Comment:
Would it be good to rename `Suggestion` to something like `SuggestionTerm` to more clearly differentiate from `SuggestionLine`?